python/src/KrigingResult_doc.i.in

```diff
@@ -290,6 +290,35 @@ Notes
 In case of fourth usage, the sequence of float is given as the concatenation of marginal variances 
 for each point in sampleX."
 
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::KrigingResult::operator()
+"Compute the expected Gaussian distribution on a point or a sample of points conditionally to the observed paths.
+
+Available usages:
+    KrigingResult(x)
+
+    KrigingResult(sampleX)
+
+Parameters
+----------
+x : sequence of float
+    The point :math:`\vect{x}` where the conditional distribution of the output has to be evaluated.
+sampleX : 2-d sequence of float
+     The sample :math:`(\vect{\xi}_1, \dots, \vect{\xi}_M)` where the conditional distribution of the output has to be evaluated (*M* can be equal to 1).
+
+Returns
+-------
+condDistribution : :class:`~openturns.Normal`
+    The conditional distribution.
+
+Notes
+-----
+
+The conditional distribution :math:`\cN(\Expect{\vect{Y}}, \Cov{\vect{Y}})` has respectively conditional mean and covariance defined 
+by `getConditionalMean` (respectively `getConditionalCovariance`)."
```

Comment:
```suggestion
The class method :meth:`getConditionalMean` returns the mean function of the distribution of the Gaussian process conditioned on the observations.

The class method :meth:`getConditionalCovariance` returns its covariance kernel."
```

I think it's better to clearly state what the distribution is conditioned on.
Besides, the `` :meth:` ` `` instruction will create links to the parts of the page that document the methods.

Reply:
There are two ways you can view a random process Y:

1. As a collection of finite-dimensional random variables.
2. As a single random variable with functional value.

Let (\Omega, F, P) be our probability space. Let us fix w \in \Omega and look at the corresponding realization.

With 1., evaluating a realization of the random process at some point x means computing the realization of the random variable Y(x) at w: Y(x)(w).
With 2., the realization Y(w) is a function that can be evaluated at some point x: Y(w)(x).

Both viewpoints are equivalent, provided you use the adequate sigma-algebras. For more details, have a look at the first chapter of [my thesis](http://theses.md.univ-paris-diderot.fr/MURE_Joseph_2_complete_20181005.pdf). 

So if you adopt the second viewpoint, the random process Y is a random variable and it has a probability distribution.

Now, if Y is a *Gaussian* random process, it has specific properties. Its distribution can be characterized by:

1. a mean function;
2. a covariance kernel.

This is analogous to the way the distribution of a Gaussian *vector* (that is a multivariate Normal distribution) can be characterized by a mean vector and a covariance matrix. In fact, a Gaussian process is essentially an infinite-dimensional Gaussian vector.

When you evaluate the process on a finite set of points (your design set), you get a Gaussian vector. The corresponding multivariate Normal distribution is essentially a multi-(but finite-)dimensional marginal of the distribution of the Gaussian process.

What happens when you condition your Gaussian process? Now its values at several points are known, so it is a different random variable with a different random distribution. Its covariance kernel is different, and its mean function is probably different too.

In my view, ``getConditionalMean`` and ``getConditionalCovariance`` are respectively accessors to that mean function and that covariance kernel. These functions can be evaluated on a Sample in order to get the mean vector and covariance matrix that would characterize the Gaussian vector obtained by evaluating the Gaussian process on this Sample.

I disagree with the "two sets of points" approach. If you evaluate the conditional Gaussian process at observed points, you get a degenerate Gaussian vector, but a Gaussian vector nonetheless, with valid mean vector and valid (singular) covariance matrix.